### ISSUE TITLE
Modify template : Fixes some settings.

### ISSUE THREAD

**Issue by daisukenishino** [OpenTouryoProject/OpenTouryo#207]
# Requirement
The record of other fixes.

### Delete unnecessary files
- \root\files\bat
- \root\files\resource\Screen

#### Delete useless comments and code
- \C#\Frameworks\Infrastructure\Business\Util\MyLiteral.cs
- ,etc.

#### Delete warning
- Public.IO.ZipBase : https://github.com/OpenTouryoProject/OpenTouryo/issues/73
- DPQuery_Tool  
Comment-out the Form3.AllColumnList.
- DaoGen_Tool
  - Comment-out the Form2.ReadDotNetTypeInfo
  - Delete none used System.Windows.Forms.Label Form2.label18

#### Reduce System.Web from Public.
Move implementation location.
- \C#\Frameworks\Infrastructure\Public\Csp → \Business\Csp
- CalculateSessionSize method  
Move to MyCmnFunction.cs from PubCmnFunction.cs
- CustomEncode
  - Add ToBase64UrlString and FromBase64UrlString method.
  - HtmlEncode and HtmlDecode method.
Change library to System.Net.WebUtility from System.Web.HttpUtility.
- ShowErrorMessage method
Add ShowErrorMessageWin and ShowErrorMessageWPF method to MyCmnFunction.

#### CustomControl
Move ImageUrl property value to config file.

#### BaseMVController
- Change implementation of MyBaseMVController such as log output method.
- Change implementation of BaseMVController
  - such as log output method.
  - to add the FxSessionAbandon method.

#### Web Forms
- The class name was changed,  
because converted from the Website to the Web application.  
Reflect the class name changes to the class header.
- I added testBlankScreenNoJs.master to run jqGrid.  
This is due to the problems was occurred between the versions.
- And testBlankScreenNoJs.master is also applied to login.aspx.
- logout.aspx is moved the implementation from click_event to page_load.  
This is because I changed logout screen to logout button.
- Delete implementation that the ping.aspx returns SessionID.  
Planed to change it to return a random number by new method.
- Changed the first character of the folder name to upper case.  
Along with that, also changed the namespace and path characters to upper case.
- Changed the bullet on the menu screen to \<ul\> - \<li\> tag.
- Resizing size of progressdialog that is collapsed by applying CSS.
- Override header and footer contentplaceholder in masterpage by the content page.
- Since the Style collapses, change the processing implemented by Response.Write.
- Deleted unnecessary references.
- Add "this" keyword. (\<%= ResolveUrl("\~/") %\> ---> \<%= this.ResolveUrl("\~/") %\>)
- Change Header to HTML5.
```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" >
```
```
<!DOCTYPE html><html>
```
- And change difinition of the style attribute according to spasification of HTML5.
- , etc.

#### MVC
- Change Crud1 processing method to Html.BeginForm.
- Change Crud2 processing method to Ajax.BeginForm.
- Change Crud1 submit method to \<inputtype="submit"・・・
- , etc.

#### SPA
- , etc.

#### RichClient
- Added the ShowErrorMessage method to the business library, and use it.
- , etc.
